doc/tutorial/argument_parser/index.md

```diff
@@ -42,54 +42,55 @@ position instead of an identifier.
-function. Since the function throws in case any errors occur, we need to wrap it into a try-catch block. Here is a 
+Before we add any of the options, flags, and positional options, we will take a look at the seqan3::argument_parser
+class itself. It is constructed by giving a program's name and passing the parameters `argc` and `argv` from main.
+Note that no command line arguments have been parsed so far, but we can now add more information to the parser.
+After adding all desired information, the parsing is triggered by calling the seqan3::argument_parser::parse member
+function. Since the function throws in case any errors occur, we need to wrap it into a try-catch block. Here is a
 first working example:
 
 \include doc/tutorial/argument_parser/basic_parser_setup.cpp
 
-There are two types of exceptions: The seqan3::design_error which indicates that the parser setup was wrong 
-(directed to the developer of the program, not the user!) and any other exception derived from 
-seqan3::argument_parser_error, which detects corrupted user input. Additionally, there are special user requests that 
-are handled by the argument parser by exiting the program via std::exit, e.g. calling `--help` that prints a help page 
+There are two types of exceptions: The seqan3::design_error which indicates that the parser setup was wrong
+(directed to the developer of the program, not the user!) and any other exception derived from
+seqan3::argument_parser_error, which detects corrupted user input. Additionally, there are special user requests that
+are handled by the argument parser by exiting the program via std::exit, e.g. calling `--help` that prints a help page
 screen.
 
 ## Design restrictions (seqan3::design_error)
 
 The argument parser checks the following restrictions and throws a seqan3::design_error if they are not satisfied:
 
 * **Long identifiers**: must be unique, more than one character long, may only contain alphanumeric characters, as well
-* as `_`, `-`, or `@`, but never start with `-`.
+as `_`, `-`, or `@`, but never start with `-`.
 * **Short identifiers**: must be unique and consist of only a single letter that is alphanumeric characters, `_` or `@`.
-* either the short or long id may be empty but not both at the same time.
+* Either the short or long id may be empty but not both at the same time.
 * Only the last positional option may be a list (see [lists](#section_list_positional_options)).
+* **Flags**: value must be false by default. Passing the flag identifier on the command line marks it as true.
 * The flag identifiers `-h`, `--help`, `--advanced-help`, `--advanced-help`, `--export-help`, `--version`, `--copyright`
-* are predefined and cannot be specified manually or used otherwise.
+are predefined and cannot be specified manually or used otherwise.
 * The seqan3::argument_parser::parse function may only be called once (per parser).
```

Comment:
Didn't see how to avoid the whitespace getting messed up. 

These lines are the actual `diff`

Reply:
Can you do a commit that only fixes the whitespaces?

See, e.g., here https://stackoverflow.com/questions/27489189/git-add-only-whitespace-changes